modules/core/src/main/scala/org/tessellation/domain/trust/storage/TrustMap.scala

```diff
@@ -0,0 +1,20 @@
+package org.tessellation.domain.trust.storage
+
+import cats.syntax.functorFilter._
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.{PublicTrust, TrustInfo}
+
+import derevo.cats.show
+import derevo.derive
+
+@derive(show)
+case class TrustMap(trust: Map[PeerId, TrustInfo], peerLabels: PublicTrustMap)
+
+object TrustMap {
+
+  val empty: TrustMap = TrustMap(Map.empty, PublicTrustMap.empty)
+
+  def toPublicTrust(trust: TrustMap): PublicTrust = PublicTrust(trust.trust.mapFilter(_.publicTrust))
```

Comment:
you can move it to `TrustMap` case class, so instead of `.map(TrustMap.toPublicTrust)` you would do `.map(_.toPublicTrust)`

Reply:
Changed to

```
@derive(show)
case class TrustMap(trust: Map[PeerId, TrustInfo], peerLabels: PublicTrustMap) {

  def toPublicTrust: PublicTrust = PublicTrust(trust.mapFilter(_.publicTrust))

}
```